### PR TITLE
changeDetailActivityToDialogFragment

### PR DESCRIPTION
**Pull request by ghilbing**

### CODE REVIEW

File: app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java [hunk 1]
```diff
@@ -99,6 +105,15 @@ private void setupTab() {
     @Override
     public void onClickListener(Book book) {
         //TODO Itent to Book Detail page
+        Log.i("Book", book.getTitle().toString());
+        Toast.makeText(this, book.getTitle().toString(), Toast.LENGTH_LONG).show();
```
Comment (m3libea) [inline app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java]: You can use the toast function that is on Base activity

File: app/src/main/java/com/fantastic/bookxchange/adapters/BooksAdapter.java [hunk 2]
```diff
@@ -78,6 +82,12 @@ public void onBindViewHolder(ViewHolder holder, int position) {
         holder.llBook.setOnClickListener(view -> {
             rowSelectedIndex=position;
             listener.onBookClickListener(book);
+            // call Detail fragment
+            Intent intent = new Intent(context, BookDetailActivity.class);
```
Comment (m3libea) [inline app/src/main/java/com/fantastic/bookxchange/adapters/BooksAdapter.java]: This is already on User activity, right?

File: app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java (hunk 1 above)
Comment (ghilbing) [reply app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java]: Ok, I forgot that

**Comment by ghilbing** [comment]
Another PR replace this one

**Pull request closed without merging**